dev/io.openliberty.jaxws.security_fat/fat/src/com/ibm/ws/jaxws/security/fat/TransportSecurityUsingDispatchClientCertTest.java

```diff
@@ -30,58 +34,38 @@
 @Mode(Mode.TestMode.FULL)
 public class TransportSecurityUsingDispatchClientCertTest extends AbstractJaxWsTransportSecurityTest {
 
-	private static final String DEFAULT_CLIENT_CERT_CONFIG = "JaxWsTransportSecurityServer/serverConfigs/defaultClientCertConfiguration.xml";
-
-	@BeforeClass
-	public static void beforeAllTests() throws Exception {
-
-		buildDefaultApps();
-		server.setServerConfigurationFile(DEFAULT_CLIENT_CERT_CONFIG);
-		lastServerConfig = "/serverConfigs/defaultClientCertConfiguration.xml";
-
-		server.startServer("TransportSecurityUsingDispatchClientCertTest.log");
+    private static final String DEFAULT_CLIENT_CERT_CONFIG = "JaxWsTransportSecurityServer/serverConfigs/defaultClientCertConfiguration.xml";
 
-		assertNotNull("Wait for the SSL port to open", server.waitForStringInLog("CWWKO0219I:.*-ssl"));
-	}
+    @BeforeClass
+    public static void beforeAllTests() throws Exception {
 
-	@AfterClass
-	public static void afterAllTests() throws Exception {
-		if (server != null && server.isStarted()) {
-			server.stopServer();
-		}
-	}
+        buildDefaultApps();
+        server.setServerConfigurationFile(DEFAULT_CLIENT_CERT_CONFIG);
+        lastServerConfig = "/serverConfigs/defaultClientCertConfiguration.xml";
 
-	@AllowedFFDC({ "java.lang.NullPointerException" })
-	@Test
-	public void testValidClientCertDispatchPOJO() throws Exception {
-		updateProviderWEBXMLFile("clientCert_provider_web.xml");
-		updateClientBndFile("bindings/validCertAlias.xml");
+        server.startServer("TransportSecurityUsingDispatchClientCertTest.log");
 
-		RequestParams params = new RequestParams("employee", TestMode.DISPATCH, "pojo", "https",
-				server.getHttpDefaultSecurePort(), "/employee/employPojoService");
-		runTest(params, "Hello, employee from SayHelloPojoService", null);
+        assertNotNull("Wait for the SSL port to open", server.waitForStringInLog("CWWKO0219I:.*-ssl"));
+    }
 
-	}
+    @AfterClass
+    public static void afterAllTests() throws Exception {
+        if (server != null && server.isStarted()) {
+            server.stopServer();
+        }
+    }
 
-	@AllowedFFDC({ "java.lang.NullPointerException" })
-	@Test
-	public void testValidClientCertDispatchStateless() throws Exception {
-		updateProviderWEBXMLFile("clientCert_provider_web.xml");
-		updateClientBndFile("bindings/validCertAlias.xml");
+    @AllowedFFDC({ "java.lang.NullPointerException" })
+    @Test
+    public void testValidClientCertDispatch() throws Exception {
```

Comment:
Can you actually move this test in to the ClientCerticateTest? I don't think there is a reason to have this as it's own separate test class, when it's doing the same thing as the ClientCertificateTest. We can just combine this test method into that class, and it should be good to go.